xdsl/dialects/riscv.py

```diff
@@ -2695,6 +2695,22 @@ class ScfgwOp(RsRsIntegerOperation):
     name = "riscv.scfgw"
 
 
+@irdl_op_definition
+class ScfgwiOp(RdRsImmIntegerOperation):
+    """
+    Write the immediate value to the Snitch stream configuration location pointed by rs
+    in the memory-mapped address space.
+
+    This is an extension of the RISC-V ISA.
```

Comment:
```suggestion
    This is part of the `Xssr' extension (https://pulp-platform.github.io/snitch/rm/custom_instructions/), an extension of the RISC-V ISA.
```
Even if the documentation is somewhat lacking on the Snitch side, I would like that we point to whatever references we have, but feel free to ignore this.